spartan/terraform/deploy-staging-testnet/values/staging-testnet-validator.yaml

```diff
@@ -11,7 +11,7 @@ global:
 
 validator:
   hostNetwork: true
-  replicaCount: 1
+  replicaCount: 4
```

Comment:
why 4 and not 8? only the first 249 attesters are actually in the set right now.